packages/component/src/Middleware/ScrollToEndButton/ScrollToEndButton.js

```diff
@@ -14,6 +14,14 @@ const ScrollToEndButton = ({ onClick }) => {
   const localize = useLocalizer();
 
   const text = localize(scrollToEndButtonBehavior === 'any' ? 'TRANSCRIPT_MORE_MESSAGES' : 'TRANSCRIPT_NEW_MESSAGES');
+  // useRef and useEffect is added to move focus on 'New messages' button when it is display on screen.
+  const buttonReference = useRef();
+
+  useEffect(() => {
+    if (buttonReference?.current) {
+      buttonReference?.current?.focus();
```

Comment:
if (buttonReference?.current) {
      buttonReference.current.focus();
      
      //since you are already checking buttonRef and current in the above line

Reply:
fixed